src/forms-opportunities/utils.js

```diff
@@ -94,41 +94,46 @@ function getFormMetrics(metricObject) {
 function convertToLowViewOpptyData(metricObject) {
   const {
     formview: { total: formViews, mobile: formViewsMobile, desktop: formViewsDesktop },
-    trafficacquisition,
+    pageview: { total: pageViews, mobile: pageViewsMobile, desktop: pageViewsDesktop },
+    // trafficacquisition,
   } = metricObject;
   return {
-    trackedFormKPIName: 'Form Views',
-    trackedFormKPIValue: formViews,
+    trackedFormKPIName: 'Form View Rate',
+    trackedFormKPIValue: Number((formViews / pageViews).toFixed(3)),
     metrics: [
       {
-        type: 'formViews',
+        type: 'formViewRate',
         device: '*',
         value: {
-          page: formViews,
+          page: Number((formViews / pageViews).toFixed(3)),
         },
       },
       {
-        type: 'formViews',
+        type: 'formViewRate',
         device: 'mobile',
         value: {
-          page: formViewsMobile,
+          page: Number((formViewsMobile / pageViewsMobile).toFixed(3)),
         },
       },
       {
-        type: 'formViews',
+        type: 'formViewRate',
         device: 'desktop',
         value: {
-          page: formViewsDesktop,
+          page: Number((formViewsDesktop / pageViewsDesktop).toFixed(3)),
         },
       },
       {
         type: 'traffic',
```

Comment:
@05rahulsingh we are using trafficacquisition of other forms oppoty as well that results in null data. can you pls fix all.